schemas/author.json

```diff
@@ -26,7 +26,14 @@
 			]
 		},
 		"country": {
-			"type": "string"
+			"type": "string",
+			"description": "ISO 3166-1 two letter code in english capital letter.",
```

Comment:
```
The author's country in a two-letter code (ISO 3166-1). See [all codes](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2#Officially_assigned_code_elements).
```

(markdown in the description is rendered correctly in the resulting API documentation)

Reply:
done.